.github/actions/add_vendordep/action.yml

```diff
@@ -0,0 +1,57 @@
+name: 'Add Vendordep'
+description: 'Adds a vendordep to the repository and creates a pull request the upstream repo'
+
+inputs:
+  token:
+    description: 'Build Buddy API token'
+    required: true
+  vendordep_file:
+    description: 'Path to the vendordep file to upload'
```

Comment:
Does this need to be absolute (since it's used inside the vendor-json-repo directory)?

Reply:
Yes